dapp-example/tests/e2e/index.spec.js

```diff
@@ -0,0 +1,71 @@
+import 'cypress-wait-until';
+import * as htsData from '../../.htsTokenAddress.json';
+
+describe('Test User Login', function() {
+  this.timeout(900000);
+
+  const resetMetamaskConnection = function() {
+    cy.disconnectMetamaskWalletFromAllDapps();
+    cy.resetMetamaskAccount();
+
+    cy.visit('http://localhost:3000');
+    cy.contains('Connect Account').click();
+    cy.switchToMetamaskWindow();
+    cy.acceptMetamaskAccess().should('be.true');
+    cy.switchToCypressWindow();
+  };
+
+  it('Connects with Metamask', function() {
+    resetMetamaskConnection();
+
+    // check the UI
+    cy.get('#showAliasBtn').should('not.be.disabled');
+    cy.get('#btnDeployContract').should('not.be.disabled');
+    cy.get('#btnReadGreeting').should('not.be.disabled');
+    cy.get('#btnUpdateGreeting').should('not.be.disabled');
+  });
+
+  it('Show alias', function() {
+    resetMetamaskConnection();
+
+    cy.get('#showAliasBtn').should('not.be.disabled').click();
+    cy.confirmMetamaskSignatureRequest();
+    cy.waitUntil(() => cy.get('#aliasField').invoke('text').should('have.length', 66));
+  });
+
+  it('Deploy contract', function() {
+    resetMetamaskConnection();
+
+    // deploy the contract
+    cy.get('#btnDeployContract').should('not.be.disabled').click();
+    cy.confirmMetamaskTransaction();
+    cy.waitUntil(() => cy.get('#btnDeployContract').should('not.be.disabled'));
+
+    // test a view call
+    cy.get('#btnReadGreeting').should('not.be.disabled').click();
+    cy.waitUntil(() => cy.get('#contractViewMsg').should('have.text', ' Result: initial_msg '));
+
+    // test a update call
+    cy.get('#updateGreetingText').type('updated_text');
+    cy.get('#btnUpdateGreeting').should('not.be.disabled').click();
+    cy.confirmMetamaskTransaction();
+    cy.waitUntil(() => cy.get('#contractUpdateMsg').should('have.text', ' Updated text: updated_text '));
+
+    // test the updated msg
+    cy.get('#btnReadGreeting').should('not.be.disabled').click();
+    cy.waitUntil(() => cy.get('#contractViewMsg').should('have.text', ' Result: updated_text '));
+  });
+
+  it('Transfer HTS token', function() {
```

Comment:
I think we're missing an HBAR transfer test in this file no?

Reply:
It is added now.